src/routes/homepage/Homepage.jsx

```diff
@@ -0,0 +1,42 @@
+import { Carousel } from "../../components";
+import { Banner } from "../../components";
+import { FeaturedCard, Footer } from "../../components";
```

Comment:
here also u can do one import instead of multiple

Reply:
noted.